app/views/layouts/listnav/_generic_object_definition_show_list_with_treeview.html.haml

```diff
@@ -1,10 +1,12 @@
 - return if @display == 'generic_objects' || @display == 'main'
 = miq_accordion_panel(_("Generic Object Classes"), true, "god") do
+  :javascript
+    ManageIQ.tree.data['#{tree.name}'] = #{tree.locals_for_render[:bs_tree]};
```

Comment:
Ugh, GOD is special, there's no `tree`, only `@tree`.

Reply:
Oh GOD WHYYY :cry: I'll fix it